01_Tutorial/01_Basics.md

```diff
@@ -55,7 +55,7 @@ If you're using the browser for this tutorial, we recommend using [unpkg](https:
 
 ```html
 <script src="https://unpkg.com/ipfs@0.35.0/dist/index.min.js"></script>
-<script src="https://www.unpkg.com/browse/orbit-db@0.25.3/dist/orbitdb.min.js"></script>
+<script src="https://www.unpkg.com/browse/orbit-db@0.25.3/dist/orbitdb.min.js" type="text/html"></script>
```

Comment:
Shouldnt this be `type="text/javascript"` ?

Reply:
nevermind, saw your comment in the OP. That's weird :P

Reply:
I agree, this would be correct, but it doesn't work as it produces:
The resource from “https://www.unpkg.com/browse/orbit-db@0.25.3/dist/orbitdb.min.js” was blocked due to MIME type (“text/html”) mismatch (X-Content-Type-Options: nosniff).